src/main/assembly/dist/install/db-tables.sql

```diff
@@ -1,5 +1,13 @@
-CREATE TABLE seed (
-    type VARCHAR(64) NOT NULL UNIQUE,
-    value BIGINT NOT NULL);
+CREATE TABLE seed(
+    type VARCHAR(64) NOT NULL,
+    value BIGINT NOT NULL,
+    PRIMARY KEY (type));
 
-GRANT INSERT, SELECT, UPDATE, DELETE ON seed TO easy_pid_generator;
+CREATE TABLE minted(
+    type VARCHAR(64) NOT NULL,
```

Comment:
`FOREIGN KEY (type) REFERENCES seed (type)`

@DANS-KNAW/easy **PLEASE TAKE NOTE:** since we do our database tests using SQLite, we have to add `PRAGMA foreign_keys = ON;` to the `*.sql` script in order for SQLite to respect the foreign key constraints!